raiden/tests/fixtures/variables.py

```diff
@@ -18,7 +18,7 @@
 # we need to use fixture for the default values otherwise
 # pytest.mark.parametrize won't work (pytest 2.9.2)
 
-DEFAULT_BALANCE = denoms.ether * 100000
+DEFAULT_BALANCE = denoms.ether * 10
```

Comment:
Why change this?

Reply:
`eth-tester` faucet doesn't have that much funds. Apart from that, 10k ether is a ridiculous amount.

Reply:
👍